src/components/Organizations/AddUserToGroupSelect/index.tsx

```diff
@@ -88,11 +88,11 @@ const AddUserToGroupSelect: FC<Props> = ({ groups, newUserState, setNewUserState
             <h4>Add user to a group</h4>
             <div className="form-box">
               <label>
-                User name: <span style={{ color: '#E30000' }}>*</span>
+              User name: <span style={{ color: '#E30000' }}>*</span>
```

Comment:
Missed this one

Reply:
no - I updated the placeholder text to match all the other instances. I'll open a separate issue for some styling elements

Reply:
User name = user email?